bench/workgen/workgen.cpp

```diff
@@ -305,6 +307,26 @@ get_dir_size_mb(const std::string &dir)
     return result / WT_MEGABYTE;
 }
 
+// 10 random characters + Null terminator
+#define DYNAMIC_TABLE_LEN 11
+/*
+ * This function generates a random table name which is alphanumeric and 10 chars in length.
```

Comment:
Should we have the length as an argument?

Reply:
I want to keep it simple and fast. Using strings or variable lengths doesn't serve us here. We would need to access and create the string character by character, and C-styled strings work best for such a case.

Reply:
It's a valid argument but concatenating strings should not be an issue and using `c_str()` whenever we are passing the string to the C layer should be fine but happy to keep it this way.